frida_tools/repl.py

```diff
@@ -127,7 +127,10 @@ def _initialize(self, parser, options, args):
         def _log(self, level, text):
             ConsoleApplication._log(self, level, text)
             if self._logfile is not None:
-                self._logfile.write(text + "\n")
+                try:
+                    self._logfile.write(text + "\n")
+                except UnicodeEncodeError:
+                    self._logfile.write(text.encode('ascii', 'ignore').decode('ascii') + '\n')
```

Comment:
```suggestion
                    self._logfile.write(text.encode('ascii', 'ignore').decode('ascii') + "\n")
```

As per our [style guide](https://github.com/frida/frida/blob/master/CONTRIBUTING.md#rule-54).

Reply:
Sorry.